src/main/java/io/appium/java_client/serverevents/ServerEvents.java

```diff
@@ -0,0 +1,23 @@
+package io.appium.java_client.serverevents;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ServerEvents {
+
+    public final List<CommandEvent> commands;
+    public final List<ServerEvent> events;
```

Comment:
this naming looks a bit unlogical to me. We have ServerEvents class, but it consist of lists with two absolutely different types: CommandEvent and ServerEvent. Perhaps, we should change the `ServerEvent` to something else to avoid confusion?

Reply:
i call it ServerEvents because it is the response to the "get server events" call. Definitely open to suggestions for a different name. Maybe instead of ServerEvent we could call it TimedEvent?